site/get-started.markdown

```diff
@@ -0,0 +1,143 @@
+3. **cabal** -> Haskell build tool -> You will use this to structure your Haskell projects, build them, run them, define dependencies, ...
+4. **Stack** -> Haskell build tool -> alternative to `cabal`
+
+
+<div class="bs-callout bs-callout-info">
+  <p>
+    <h4>cabal and Stack -> which one should I install?</h4>
+    cabal is the original build tool, while Stack was created as an alternative to cabal a time ago to solve some of the problems that cabal had and provide a more user-friendly experience. In the meantime, cabal solved most of those issues (including "cabal hell") and caught up with Stack regarding user experience, so the main difference between them at the moment is how they resolve dependencies, which for you as a beginner isn't a concern. Therefore, both are a good choice, and if not sure, you can install both and then use whatever the learning resources you will later use will point you to.
+  </p>
+</div>
+
+To confirm it is all installed correctly, you can run the following commands:
+```
+> ghc --version
+> haskell-language-server-wrapper --version
+> cabal --version                               # If you installed cabal
+> stack --version                               # If you installed Stack
+```
+
+### Editor
+HLS (Haskell Language Server), which you just installed via GHCup, is the one that brings all the cool IDE features to editors, so as long as your editor has a decent Haskell language extension that utilizes HLS, you are good.
+
+The most straightforward option is to go with **Visual Studio Code** - just install [Haskell extension](https://marketplace.visualstudio.com/items?itemName=haskell.haskell) and you are all set, it should work out of the box and use your installation of HLS.
+
+Of other editors that have good Haskell extensions, the most popular ones are Vim and Emacs.
+
+## Running first lines of code
+
+We have everything set up, let's use it!
+
+`GHC` brings an interactive interpreter called `GHCi` together with it, which is great for playing with Haskell and trying things out, so let's give it a spin.
+So even though Haskell is a compiled language, it comes with an interpreter too -> how cool is that?
+
+Run `ghci`, which should start a new prompt for you.
+
+Let's do a simple calculation to check Haskell's computing capabilities:
+```
+> 6 + 3^2 * 4
+42
+```
+Hey, they call Haskell lazy, but that was quick!
+
+42 is a nice even number, but what about the even numbers after it? Let's get the first 10 even numbers after 42.
+```
+> take 10 $$ filter even [43..]
+[44,46,48,50,52,54,56,58,60,62]
+```
+
+What is the sum of those?
+```
+> sum it
+530
+```
+**NOTE**: We used a special feature of GHCi here, which is a special `it` variable that remembers the result of the last expression.
+
+Great, you got the first taste of Haskell! Now let's get to running a real program.
+
+## Writing your first Haskell program
+
+In your editor, create a new file named `hievb.hs`.
```

Comment:
```suggestion
In your editor, create a new file named `hello.hs`.
```

Reply:
Again, same as above, I don't mind making a change if it makes sense, but I can't make any kind of judgement without an argument being provided.

Reply:
The word hello exists, is descriptive and has meaning. hievb is not a word and I couldn't tell what it means without thinking about it a little, and then saying to myself "why not just hello?".